main/admin/user_edit.php

```diff
@@ -211,6 +211,7 @@ function confirmation(name) {
 $status[STUDENT] 		= get_lang('Learner');
 $status[DRH] 			= get_lang('Drh');
 $status[SESSIONADMIN] 	= get_lang('SessionsAdmin');
+$status[STUDENT_BOSS] 	= get_lang('RoleStudentBoss');
```

Comment:
I will add it, but just to make the point, the RoleStudentBoss was not added to translate.chamilo.org